gini/gini-vision-lib-android
# Finalize No Results Screen

**Note:** Filter for `java` when reviewing, a lot of images were updated and added.

Updated the UI of the No Results Screen. Since analysis can complete also while in the Review Screen the No Results Screen can be shown from there, too.

Updated the example apps to showcase the usage of the No Results Screen.

### How to test
Run the example apps and use images and pdfs that don't contain Pay 5 information to try out the following scenarios:
* Take picture: should show the No Results Screen from Review Screen (wait a little for the analysis to complete) and Analysis Screen
* Import picture: same as above
* Import PDF: should NOT show the No Results Screen (might show a different one in the Screen API Example App)
* Open with image: should show the No Results Screen from Review Screen (wait a little for the analysis to complete) and Analysis Screen WITHOUT the bottom button to go back to the camera
* Open with PDF: same as Import PDF

### Merging
Automatic.

### Ticket 
Issue #86
